### PR TITLE
[Communication] - Documentation - Update docs

### PR DESCRIPTION
**Pull request by JoshuaLai**

### CODE REVIEW

**Review by sacheu — CHANGES_REQUESTED** [review]

**Comment by sacheu** [comment]
In general, in some places, breakin the text into 2 lines make the comment harder to read.

**Comment by JoshuaLai** [comment]
> In general, in some places, breakin the text into 2 lines make the comment harder to read.

Yes I did that on purpose to fix warnings. There was a bunch of warnings the linter was complaining about.

**Comment by sacheu** [comment]
> > In general, in some places, breakin the text into 2 lines make the comment harder to read.
> 
> Yes I did that on purpose to fix warnings. There was a bunch of warnings the linter was complaining about.

I guess I don't understand why linter was complaining about. Since the line was not that long originally.

**Comment by JoshuaLai** [comment]
Just double checking, but yes the liner was complaining that the character limit is 120 and in some cases we were just a little bit over (124 characters) or very over (240 characters) so the goal was to just clean these up

**Comment by sacheu** [comment]
Is it possible to change the text to shorter so that they can be in same line? e.g. 
"and" -> "&"
"Set this to true " -> "Set to true"

File: sdk/communication/AzureCommunication/Source/Identifiers.swift [hunk 1]
```diff
@@ -27,7 +27,9 @@
 import Foundation
 
 /**
- Common Communication Identifier protocol for all Azure Communication Services. All Communication Identifiers conform to this protocol.
+ Common Communication Identifier protocol for all
+ Azure Communication Services.
```
[the remark this answers is not in the document]
Comment (sacheu) [reply sdk/communication/AzureCommunication/Source/Identifiers.swift:31]: and seems like '. All Communication Identifiers conform to this protocol.' is removed.

File: sdk/communication/AzureCommunication/Source/Authentication/AutoRefreshTokenCredential.swift [hunk 2]
```diff
@@ -60,10 +62,12 @@ internal class AutoRefreshTokenCredential: CommunicationTokenCredentialProviding
     }
 
     /**
-     Retrieve an access token from the cache, or from the `tokenRefresher` if the token is not in the cache or is expired.
+     Retrieve an access token from the cache, or from the `tokenRefresher`
+     if the token is not in the cache or is expired.
 
      - Parameter completionHandler: Closure that accepts an optional `AccessToken` or optional `Error` as parameters.
-     `AccessToken` returns  a token and an expiry date if applicable. `Error` returns `nil` if the current token can be returned.
+     `AccessToken` returns  a token and an expiry date if applicable.
```
Comment (sacheu) [inline sdk/communication/AzureCommunication/Source/Authentication/AutoRefreshTokenCredential.swift:69]: there is an extra space between 'returns' and 'a'

**Comment by sacheu** [comment]
And in the Chat generated code, there is `// swiftlint:disable line_length` 
I wonder if we should do this instead of breaking 1 sentence comment into 2 lines 

 https://github.com/Azure/azure-sdk-for-ios/blob/41039cc13f33b515d2fe353b478707df6840f140/sdk/communication/AzureCommunicationChat/Source/Generated/Models/UpdateChatMessageRequest.swift

**Comment by sacheu** [comment]
And in the Chat generated code, there is `// swiftlint:disable line_length` 
I wonder if we should do this instead of breaking 1 sinto 2 lines 

 https://github.com/Azure/azure-sdk-for-ios/blob/41039cc13f33b515d2fe353b478707df6840f140/sdk/communication/AzureCommunicationChat/Source/Generated/Models/UpdateChatMessageRequest.swift

File: .swiftlint.yml [hunk 3]
```diff
@@ -1,2 +1,5 @@
 disabled_rules: # rule identifiers to exclude from running
   - large_tuple
+
```
Comment (JoshuaLai) [inline .swiftlint.yml:3]: @tjprescott @sacheu making a new addition to swiftlint line length rule to ignore comments.

File: sdk/communication/AzureCommunication/Source/Authentication/AutoRefreshTokenCredential.swift [hunk 4]
```diff
@@ -63,7 +64,8 @@ internal class AutoRefreshTokenCredential: CommunicationTokenCredentialProviding
      Retrieve an access token from the cache, or from the `tokenRefresher` if the token is not in the cache or is expired.
 
      - Parameter completionHandler: Closure that accepts an optional `AccessToken` or optional `Error` as parameters.
-     `AccessToken` returns  a token and an expiry date if applicable. `Error` returns `nil` if the current token can be returned.
+     `AccessToken` returns a token and an expiry date if applicable.
+     `Error` returns `nil` if the current token can be returned.
```
Comment (sacheu) [inline sdk/communication/AzureCommunication/Source/Authentication/AutoRefreshTokenCredential.swift:68]: Can this change revert now?

File: sdk/communication/AzureCommunication/Source/Authentication/AutoRefreshTokenCredential.swift [hunk 5]
```diff
@@ -29,7 +29,8 @@
 #endif
 import Foundation
 /**
- The Azure Communication Services User token credential. This class is used to cache/refresh the access token required by Azure Communication Services.
+ The Azure Communication Services User token credential.
+ This class is used to cache/refresh the access token required by Azure Communication Services.
```
Comment (sacheu) [inline sdk/communication/AzureCommunication/Source/Authentication/AutoRefreshTokenCredential.swift:33]: Can this change revert now?

File: sdk/communication/AzureCommunication/Source/Authentication/CommunicationTokenCredential.swift [hunk 6]
```diff
@@ -33,7 +33,8 @@ public typealias CommunicationTokenCompletionHandler = (CommunicationAccessToken
 public typealias TokenRefreshOnCompletion = (String?, Error?) -> Void
 
 /**
- The Azure Communication Services User token credential. This class is used to cache/refresh the access token required by Azure Communication Services.
+ The Azure Communication Services User token credential.
+ This class is used to cache/refresh the access token required by Azure Communication Services.
```
Comment (sacheu) [inline sdk/communication/AzureCommunication/Source/Authentication/CommunicationTokenCredential.swift:37]: Can this change revert now?

File: sdk/communication/AzureCommunication/Source/Identifiers.swift [hunk 7]
```diff
@@ -27,7 +27,8 @@
 import Foundation
 
 /**
- Common Communication Identifier protocol for all Azure Communication Services. All Communication Identifiers conform to this protocol.
+ Common Communication Identifier protocol for all
+ Azure Communication Services. All Communication Identifiers conform to this protocol.
```
Comment (sacheu) [inline sdk/communication/AzureCommunication/Source/Identifiers.swift:31]: Can this change revert now?

File: sdk/communication/AzureCommunication/Source/Identifiers.swift [hunk 8]
```diff
@@ -98,7 +99,8 @@ import Foundation
     /**
      Creates a MicrosoftTeamsUserIdentifier object
      - Parameter userId: Id of the Microsoft Teams user. If the user isn't anonymous, the id is the AAD object id of the user.
-     - Parameter isAnonymous: Set this to true if the user is anonymous, for example when joining a meeting with a share link.
+     - Parameter isAnonymous: Set this to true if the user is anonymous,
+     for example when joining a meeting with a share link.
```
Comment (sacheu) [inline sdk/communication/AzureCommunication/Source/Identifiers.swift:103]: Can this change revert now?

**Review by sacheu — APPROVED** [review]
Looks good to me. I think some changes can be reverted now.

File: sdk/communication/AzureCommunication/Source/Authentication/CommunicationTokenCredential.swift [hunk 9]
```diff
@@ -66,7 +67,8 @@ public typealias TokenRefreshOnCompletion = (String?, Error?) -> Void
     /**
      Retrieve an access token from the credential.
      - Parameter completionHandler: Closure that accepts an optional `AccessToken` or optional `Error` as parameters.
-     `AccessToken` returns a token and an expiry date if applicable. `Error` returns `nil` if the current token can be returned.
+     `AccessToken` returns a token and an expiry date if applicable.
+     `Error` returns `nil` if the current token can be returned.
```
Comment (sacheu) [inline sdk/communication/AzureCommunication/Source/Authentication/CommunicationTokenCredential.swift:71]: Can this change revert now?

**Pull request merged**